android/settings.gradle

```diff
@@ -10,7 +10,7 @@ include 'liferay-westeros-viewset'
 project(':liferay-westeros-viewset').projectDir = new File(settingsDir, '/viewsets/westeros')
 project(':liferay-westeros-viewset').name = 'liferay-westeros-viewset'
 
-include ':addbookmarkscreenlet'
+include ':addbookmarkscreenlet', ':westerosemployees-hybrid'
```

Comment:
pass android lint to see it there are issues in your module, feel free to ask any doubts whatsoever to me or victor :)